tests/tests.js

```diff
@@ -4,6 +4,23 @@ var assert = chai.assert;
 
 describe('Initialization', function() {
 
+  describe('Viewer', () => {
+    it('initializes correctly with canvas passed in as argument', () => {
```

Comment:
```suggestion
  describe('Viewer', function() {
    it('initializes correctly with canvas passed in as argument',  function() {
```
Keep same function style